bazel/external/jinja.BUILD

```diff
@@ -1,8 +1,60 @@
 licenses(["notice"])  # Apache 2
 
+# Jinja2 2.11 sources are stored in 'src/jinja2' compared to 2.10's 'jinja2'.
+# We cannot make a py_library naively with glob, because then the result package would be
+# 'src.jinja2', instead of 'jinja2'.
+# Cannot work around this with stuff like `import src.jinja2` because it looks like jinja2 calls
+# its own packages via reflection.
+# So solving it the trivial way, moving the source files up one directory.
+# If http_archive's 'strip_prefix' attribute could strip multiple segments instead of just one,
+# we could have just ignored 'jinja-2.x.y/src' instead of doing all this trickery here.
+
+filegroup(
+    name = "jinja2_sources",
+    srcs = glob(["src/jinja2/*.py"]),
+)
+
+genrule(
+    name = "move_sources_up_one_level",
+    srcs = ["//:jinja2_sources"],
+    outs = [
```

Comment:
I couldn't figure this out, I wish I could do something like
outs = `[f.replace('src/jinja2', 'jinja2') for f in //:jinja2_sources]`